<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menu</class>
 <widget class="QWidget" name="menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>bienvenue !!!</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/new/prefix1/C:/Users/hadhoud/Downloads/android takeover (1).jpg</normaloff>:/new/prefix1/C:/Users/hadhoud/Downloads/android takeover (1).jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>761</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="Statistiques" stdset="0">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Ajouter/Modifier un client</string>
    </attribute>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>M</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>80</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Mme</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_3">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>80</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Melle</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>civilité</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>CIN</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>nom </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>180</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>prenom</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>240</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>email</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>210</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>age</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>300</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>type de client </string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>300</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Devloppeur</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>300</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Entreprise</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>120</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>150</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>180</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active/>
       <inactive/>
       <disabled/>
      </palette>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>240</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_3">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>210</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>270</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Activité</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_12">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>270</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="ajouter">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>330</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>image/1450942_1415772515321220_605390347_s.jpg</normaloff>image/1450942_1415772515321220_605390347_s.jpg</iconset>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="modification">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>330</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>image/1471273_1415772591987879_960258490_s.jpg</normaloff>image/1471273_1415772591987879_960258490_s.jpg</iconset>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-20</y>
       <width>771</width>
       <height>431</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/new/prefix1/C:/Users/hadhoud/Desktop/Untitled-1.png</pixmap>
     </property>
    </widget>
    <zorder>label_10</zorder>
    <zorder>radioButton</zorder>
    <zorder>radioButton_2</zorder>
    <zorder>radioButton_3</zorder>
    <zorder>label_8</zorder>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>label_4</zorder>
    <zorder>label_5</zorder>
    <zorder>label_6</zorder>
    <zorder>label_7</zorder>
    <zorder>checkBox</zorder>
    <zorder>checkBox_2</zorder>
    <zorder>lineEdit</zorder>
    <zorder>lineEdit_2</zorder>
    <zorder>lineEdit_3</zorder>
    <zorder>lineEdit_4</zorder>
    <zorder>spinBox_3</zorder>
    <zorder>label_20</zorder>
    <zorder>lineEdit_12</zorder>
    <zorder>ajouter</zorder>
    <zorder>modification</zorder>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Afficher / Imprimer La liste des clients</string>
    </attribute>
    <widget class="QTableView" name="tableView_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>481</width>
       <height>281</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/new/prefix1/C:/Users/hadhoud/Desktop/2013-Land-Rover-Range-Rover.jpg);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="load_table_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>340</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton#load_table {
     background-color: red;
     border-width: 2px;
     border-color: beige;
     min-width: 10em;
     padding: 6px;
 }

</string>
     </property>
     <property name="text">
      <string>Actualiser la liste des clients</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>image/581998_1415772538654551_1881795736_t.jpg</normaloff>image/581998_1415772538654551_1881795736_t.jpg</iconset>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>761</width>
       <height>411</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/new/prefix1/C:/Users/hadhoud/Desktop/Untitled-1.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>340</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>imprimer La Liste des clients</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>C:/Users/hadhoud/Desktop/pdf.jpg</normaloff>C:/Users/hadhoud/Desktop/pdf.jpg</iconset>
     </property>
    </widget>
    <zorder>label_12</zorder>
    <zorder>tableView_2</zorder>
    <zorder>load_table_2</zorder>
    <zorder>pushButton_6</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Supprimer un client </string>
    </attribute>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>90</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>chercher le  CIN qui on supprimer</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_10">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>130</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="delete_2">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>170</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>image/1459083_1415772488654556_404435789_t.jpg</normaloff>image/1459083_1415772488654556_404435789_t.jpg</iconset>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>20</y>
       <width>20</width>
       <height>321</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="load_table">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>340</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Actualiser la liste des clients</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>image/581998_1415772538654551_1881795736_t.jpg</normaloff>image/581998_1415772538654551_1881795736_t.jpg</iconset>
     </property>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>471</width>
       <height>311</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/new/prefix1/C:/Users/hadhoud/Desktop/2013-Land-Rover-Range-Rover.jpg);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-60</y>
       <width>1331</width>
       <height>611</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/new/prefix1/C:/Users/hadhoud/Desktop/Untitled-1.png</pixmap>
     </property>
    </widget>
    <zorder>label_14</zorder>
    <zorder>label_9</zorder>
    <zorder>label</zorder>
    <zorder>lineEdit_10</zorder>
    <zorder>delete_2</zorder>
    <zorder>line</zorder>
    <zorder>load_table</zorder>
    <zorder>tableView</zorder>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>contacter  un client</string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit_11">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>60</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>60</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>CIN  Client:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="label_19">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>230</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Envoyer</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>C:/Users/hadhoud/Desktop/ExempleEmploye/ok.jpg</normaloff>C:/Users/hadhoud/Desktop/ExempleEmploye/ok.jpg</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>100</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Afficher</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/new/prefix1/image/1426204_1415772738654531_1287283233_s.jpg</normaloff>:/new/prefix1/image/1426204_1415772738654531_1287283233_s.jpg</iconset>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-20</y>
       <width>961</width>
       <height>451</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/new/prefix1/C:/Users/hadhoud/Desktop/Untitled-1.png</pixmap>
     </property>
    </widget>
    <widget class="QListView" name="tableView_4">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>160</y>
       <width>256</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <zorder>label_13</zorder>
    <zorder>lineEdit_11</zorder>
    <zorder>label_17</zorder>
    <zorder>label_19</zorder>
    <zorder>pushButton_2</zorder>
    <zorder>tableView_4</zorder>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Statistiques</string>
    </attribute>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-20</y>
       <width>961</width>
       <height>451</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/new/prefix1/C:/Users/hadhoud/Desktop/Untitled-1.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>310</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Le produit le moins chére</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/new/prefix1/image/1393886_1415772731987865_1761438597_t.jpg</normaloff>:/new/prefix1/image/1393886_1415772731987865_1761438597_t.jpg</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>310</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Le produit le plus chére</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/new/prefix1/image/1460017_1415772598654545_2058949048_n.jpg</normaloff>:/new/prefix1/image/1460017_1415772598654545_2058949048_n.jpg</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>310</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Le produit plus vendus</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/new/prefix1/image/1460096_1415772768654528_694278824_n.jpg</normaloff>:/new/prefix1/image/1460096_1415772768654528_694278824_n.jpg</iconset>
     </property>
    </widget>
    <widget class="QTableView" name="tableView_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>481</width>
       <height>231</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix3/amine/depositphotos_2609430-Business-finance-chart-graph-diagram.jpg);</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>-160</x>
     <y>-340</y>
     <width>1121</width>
     <height>631</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/new/prefix1/C:/Users/hadhoud/Downloads/android takeover (1).jpg</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_11">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>0</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/image/deconnexion (1).png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_11</zorder>
  <zorder>tabWidget</zorder>
  <zorder>pushButton_11</zorder>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
